core/templates/dev/head/components/ParamChangesEditorDirective.js

```diff
@@ -202,7 +202,9 @@ oppia.directive('paramChangesEditor', [function() {
 
           explorationParamSpecsService.saveDisplayedValue();
           $scope.paramChangesService.saveDisplayedValue();
-          $scope.postSaveHook();
+          if ($scope.postSaveHook) {
```

Comment:
Since you removed the parameter for postSaveHook in the directive HTML below, is this ever going to be set?